tests/OK/inductive_type.lp

```diff
@@ -0,0 +1,11 @@
+inductive nat : TYPE ≔
+ | z    : nat
+ | succ : nat → nat
+
+inductive list ≔
+ | nil  : list
+ | cons : a → list → list
+
+symbol nat : TYPE
+
+definition test (x:nat) ≔ x
```

Comment:
Once inductive is implemented, more interesting tests should be given instead like buillding a nat or a list to check that constructors and types are indeed correctly added in the signature: `assert succ z : nat`.

Reply:
It's a mistake. I deleted it. 
Now, the file pretty.ml is OK (it wasn't the case before).